sdk/src/main/scala/com/stratio/sparkta/sdk/Operator.scala

```diff
@@ -45,7 +45,8 @@ with Ordered[Operator] with TypeConversions {
 
   def writeOperation: WriteOp
 
-  protected val inputField = properties.getOptionAs[String]("inputField")
+  //  protected val inputField = properties.getOptionAs[String]("inputField")
+  val inputField = if (properties.contains("inputField")) Some(properties.getString("inputField")) else None
```

Comment:
getString has a default value.
Could this line be replaced by `val inputField = properties.getString("inputField", None)` ?